web/src/app/admin/assistants/AssistantEditor.tsx

```diff
@@ -531,13 +531,13 @@ export function AssistantEditor({
                     </TooltipProvider>
                   </div>
                   <p className="my-1 text-text-600">
-                    You assistant will use your system default (currently{" "}
-                    {defaultModelName}) unless otherwise specified below.
+                    Your assistant will use your user&apos;s set default unless
```

Comment:
nit: `Your assistant will use your user's` -> `Your assistant will use the user's`. A lot of "your"s, feels a bit of confusing. Also would be nice to add something like "For you, this would be `X`". This is especially helpful when individuals (not admins) are creating assistants.